fix(editor): Fix various element-plus styles

OlegIvaniv opened the pull request:
This PR fixes following styles
- Icons
- Default select size(no more medium in element-plus)
- Tags sizes inside select--default(medium)
- Centered layout

OlegIvaniv committed:
fix(editor): Fix various element-plus styles

Signed-off-by: Oleg Ivaniv <me@olegivaniv.com>

OlegIvaniv committed:
Remove debugging code

Signed-off-by: Oleg Ivaniv <me@olegivaniv.com>

MiloradFilipovic commented on packages/design-system/src/css/select.scss:
```diff
@@ -165,4 +165,9 @@
 	}
+	&--default {
+		.el-tag {
+			padding: 2px 12px;
```
Can we use our spacing tokens (`--spacing-5xs` and `--spacing-xs`) instead of hard-coded pixel values?

MiloradFilipovic commented on packages/editor-ui/src/components/TagsDropdown.vue:
```diff
@@ -253,8 +253,7 @@ export default defineComponent({
 
-	:deep(.el-select) {
-		.el-select__tags {
+		.el-tag {
```
This now has unmatched curly brackets:
![image](https://github.com/n8n-io/n8n/assets/2598782/8318c1aa-054d-4764-aa1f-22236e87dde6)

OlegIvaniv replied:
Ops, I deleted one line too many before committing. Thanks!

MiloradFilipovic commented on packages/editor-ui/src/n8n-theme.scss:
```diff
@@ -69,6 +69,13 @@
 		color: $custom-dialog-text-color;
 	}
+	.el-message-box-icon {
+		width: 24px;
```
Nit: Can we use `--spacing-l`?

MiloradFilipovic reviewed (commented):
Thanks for addressing the layout issue 🙌 
Left some minor comments and one css error that needs fixing.

OlegIvaniv committed:
Address PR comments

Signed-off-by: Oleg Ivaniv <me@olegivaniv.com>

OlegIvaniv commented:
> Thanks for addressing the layout issue 🙌 Left some minor comments and one css error that needs fixing.

Thanks for the review, @MiloradFilipovic! I've addressed your points. Can you have another look, please?

MiloradFilipovic reviewed (approved):
Looks good. Thanks for addressing.
